frontend/src/metabase/querying/filters/components/FilterValuePicker/ListValuePicker/ListValuePicker.tsx

```diff
@@ -70,24 +80,34 @@ function CheckboxListPicker({
         autoFocus={autoFocus}
         onChange={handleInputChange}
       />
-      <Checkbox.Group value={selectedValues} onChange={onChange}>
-        {visibleOptions.length > 0 ? (
-          <Stack>
-            {visibleOptions.map(option => (
-              <Checkbox
-                key={option.value}
-                value={option.value}
-                label={option.label}
-              />
-            ))}
-          </Stack>
-        ) : (
-          <Stack c="text-light" justify="center" align="center">
-            <Icon name="search" size={40} />
-            <Text c="text-medium" fw="bold">{t`Didn't find anything`}</Text>
-          </Stack>
-        )}
-      </Checkbox.Group>
+      {visibleOptions.length > 0 ? (
+        <Stack>
+          <Checkbox
+            variant="stacked"
+            label={isAll ? `Select none` : t`Select all`}
+            checked={isAll}
+            indeterminate={!isAll && !isNone}
+            fw="bold"
```

Comment:
There's no designs in the product doc, but the column picker renders the "Select all/none" option as regular text (ie. not bold) and with a separator. Should we follow the same pattern here?

<img width="317" alt="Screenshot 2024-09-12 at 14 15 46" src="https://github.com/user-attachments/assets/08518ae7-322d-4267-9994-18345a14f7d2">

Reply:
Nope, we explicitly don't want to do it the same here. See the slack thread in the description